clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_pipeline.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+
+
+import clinica.pipelines.engine as cpe
+
+
+# Use hash instead of parameters for iterables folder names
+# Otherwise path will be too long and generate OSError
+from nipype import config
+cfg = dict(execution={'parameterize_dirs': False})
+config.update_config(cfg)
+
+
+class DeepLearningPrepareData(cpe.Pipeline):
+    """Deeplearning prepare data - MRI in nifty format are transformed into
+    Pytorch tensors. The transformation is applied to: the whole volume, a
+    selection of 3D patches, or slices extracted from the 3D volume.
+
+    Warnings:
+        - A warning.
+
+    Todos:
+        - [x] A filled todo item.
+        - [ ] An ongoing todo item.
+
+    Returns:
+        A clinica pipeline object containing the Deeplearning prepare data pipeline.
+
+    Raises:
+
+    """
+
+    def check_custom_dependencies(self):
+        """Check dependencies that can not be listed in the `info.json` file."""
+        pass
+
+    def get_input_fields(self):
+        """Specify the list of possible inputs of this pipeline.
+
+        Returns:
+            A list of (string) input fields name.
+        """
+
+        return ['t1w']  # Fill here the list
```

Comment:
Since it will be extended to other pipelines like `pet-volume`, I would use a wording more general e.g. `input_nifti`. If you find better naming convention, don't hesitate to change.

BTW, you can remove `# Fill here the list`

Reply:
Done!